packages/block-library/src/verse/edit.js

```diff
@@ -21,6 +21,11 @@ export default function VerseEdit( {
 	mergeBlocks,
 } ) {
 	const { textAlign, content } = attributes;
+	const blockWrapperProps = useBlockWrapperProps( {
+		className: classnames( className, {
+			[ `has-text-align-${ textAlign }` ]: textAlign,
+		} ),
```

Comment:
Passing in `className` here is unnecessary.
```suggestion
		className: classnames( {
			[ `has-text-align-${ textAlign }` ]: textAlign,
		} ),
```